helm/sealed-secrets/values.yaml

```
@@ -360,6 +360,9 @@ metrics:
     ## scrapeTimeout: 10s
     ##
     scrapeTimeout: ""
+    ## @param metrics.serviceMonitor.honorLabels Specify if ServiceMonitor endPoints will honor labels
+    ##
+    honorLabels: false
```

```suggestion
    honorLabels: true
```

The default is true, so we should keep it to avoid backwards incompatibilities.